bundler.d/puma.rb

```diff
@@ -0,0 +1 @@
+gem 'puma', '3.11.4'
```

Comment:
This should be in `group :puma` so users can turn it on or off via bundler. In RPM deployments, we need to do packaging change and do subpackage so installing `foreman-proxy-puma` will actually grab the dependency. Also I'd probably add '~> 3.11' requirement, not sure about the gemspec syntax tho.